src/k8s/pkg/snap/snap_test.go

```diff
@@ -0,0 +1,89 @@
+package snap
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	. "github.com/canonical/k8s/pkg/snap/mock/runner"
```

Comment:
Avoid `.` imports, as it makes hard to track down where things are coming from

Reply:
`gomega` is in fact an exception to this rule, because it makes writing test assertions easier.